test/performance/search/views/view_minimiser_hash_benchmark.cpp

```diff
@@ -98,7 +98,7 @@ void compute_minimisers(benchmark::State & state)
             // Use random seed to randomise order on forward strand.
             auto forward = seq | seqan3::views::kmer_hash(shape)
                          | std::views::transform(
-                               [seed](uint64_t const i)
+                               [](uint64_t const i)
                                {
                                    return i ^ seed;
                                });
```

Comment:
how is `seed` available now?

Reply:
It's implicitly captured because it's const.